test/e2e/serial/config/fixture.go

```diff
@@ -91,5 +94,14 @@ func NewFixtureWithOptions(nsName string, options e2efixture.Options) (*E2EConfi
 	cfg.SchedulerName = cfg.NROSchedObj.Status.SchedulerName
 	klog.Infof("detected scheduler name: %q", cfg.SchedulerName)
 
+	initialPFPState, err = intnrt.GetCurrentPFPState(cfg.Fixture.Client, context.TODO())
```

Comment:
does this belong to the Fixture or the E2Econfig?
if it belongs to the Fixture, it is recorded multiple times during the testsuite lifecycle, each per group of test (belonging to the same fixture!)
If it belongs to the E2EConfig, it is recorded once when all the testsuite runs, before any Fixture runs

Reply:
It actually belongs to the e2econfig, that's the idea to record the pfp in before suite. Thanks for catching this, will fix it.